sim/rogue/talents.go

```diff
@@ -313,17 +313,18 @@ func (rogue *Rogue) registerBladeFlurryCD() {
 			rogue.MultiplyMeleeSpeed(sim, 1/1.2)
 		},
 		OnSpellHitDealt: func(aura *core.Aura, sim *core.Simulation, spell *core.Spell, result *core.SpellResult) {
-			isFoKOH := false
+			bfEligible := true
 
-			if spell.ActionID.SpellID == 409240 && spell.ActionID.Tag == 2 {
-				isFoKOH = true
+			//Checks for FoK Offhand and 2P TAQ Set Piece Extra Hits.
+			if (spell.ActionID.SpellID == 409240 && spell.ActionID.Tag == 2) || spell.ActionID.SpellID == 1213754 {
```

Comment:
Could you potentially replace the SpellID checks here with spell codes? I know that these were already here so if it's a pain then don't worry about it for now

Reply:
I dont think we have SpellCodes for FoK because its only checking the Action ID tag which is the offhand.  So will leave this one as is.